src/screens/dashboard/dashboard.tsx

```diff
@@ -140,19 +148,60 @@ export class DashboardScreenComponent extends React.Component<
         }
     }
 
-    public async componentDidMount() {
+    public componentDidMount() {
         this.props.navigation.setParams({
             setDashboardMenuBottomSheet: this.setDashboardMenuBottomSheet
         });
     }
 
+    public componentDidUpdate(prevProps: IReduxProps) {
+        if (this.props.selectedBlockchain !== prevProps.selectedBlockchain) {
+            const found =
```

Comment:
`found` is too confusing as a name for this context :)) you are basically verifying if something is not found ...

Reply:
Forgot about this :)) Sorry, fixing :)

Reply:
also you can use indexOf to check if an element is in an array :P

Reply:
yes, much easier with `indexOf`, fixed :D